src/node_modules/app/utils/builderUtils.js

```diff
@@ -0,0 +1,129 @@
+import * as _ from 'lamb';
```

Comment:
I'd rename this file as `src/node_modules/app/components/elementary/TypedField.utils.js`